scripts/proteinInteractionEBI/README.md

```diff
@@ -0,0 +1,72 @@
+# Scripts for importing ontology dataset from the European Bioinformatics Institute (EMBL-EBI)
+
+```
```

Comment:
nit: I would drop this hierarchy. Gets out of date when you add files, and the browsing github we get to see the same structure :)